CHANGES.rst

```diff
@@ -8,45 +8,194 @@ https://hyperspy.readthedocs.io/en/latest/user_guide/changes.html
 
 .. towncrier-draft-entries:: |release| [UNRELEASED]
 
+.. towncrier release notes start
+
+Hyperspy 2.0rc0 (2023-11-10)
+============================
+
+Release Highlights
+------------------
+- Hyperspy has split off some of it's functionality!
+  - `RosettaSciIO<https://hyperspy.org/rosettasciio/changes.html>`_: A package for reading
+     and writing scientific data files.
+  - `Exspy<https://exspy.readthedocs.io/en/latest/>`_: A package for EELS and EDS analysis.
+  - `HoloSpy<https://holospy.readthedocs.io/en/latest/>`_: A package for analysis of
+    (off-axis) electron holography data.
+- Support for Generalised Oscillator Strengths (GOS) implemented in
```

Comment:
Doesn't this belong to the exSpy `CHANGES.rst` now? The same goes for the IO entries below that maybe should go to RosettaSciIO. Maybe we could simply add an entry pointing to the external changelogs just for this release.

Reply:
I think they should be moved over there but I don't think it hurts to have it documented both places.  I think people are used to looking ay hyperspy for changes and less likely to look at exspy for changes so anything to ease that transition might be nice.

Reply:
Went for adding the links to the respective release notes.